Review and update README instructions (#13)
apache/openwhisk-vscode-extension: README.md

- `Create a wskdeploy project`: creates a wskdeploy project

## How to debug in your local environment

### Requirements

- [vscode](https://code.visualstudio.com/) >= 1.41.0
- [Node.js](https://nodejs.org/en/download/) >= 12.x

> - Have a OpenWhisk deployment available, can get set up by following [guide](https://openwhisk.apache.org/documentation.html#pre-requisites)

Reviewer: ```suggestion
- Have an OpenWhisk deployment available, which you set up by following [guide](https://openwhisk.apache.org/documentation.html#pre-requisites) or by using one of the managed OpenWhisk offerings (e.g., from IBM, Adobe or Nimbella).
```

Author: Added this in commit be186d8

### Set up

1. Clone the repository:

    ```bash
    git clone https://github.com/apache/openwhisk-vscode-extension.git
    code ./openwhisk-vscode-extension # Open openwhisk-vscode-extension in VS Code
    ```

> 2. In terminal execute command `npm install` to install extension dependencies

Reviewer: ```suggestion
2. In your terminal, execute the command `npm install` to install extension dependencies
```

Author: Added this in commit be186d8

> 3. In terminal execute command `npm run webpack-dev` to start up the extension builder in watch mode (allows for making changes to files and rebuilding automatically)

Reviewer: ```suggestion
3. In your terminal, execute the command `npm run webpack-dev` to start up the extension builder in watch mode (allows for making changes to files and rebuilding automatically)
```

Author: Added this in commit be186d8
